product_pricelist_supplierinfo/models/product_template.py

```diff
@@ -48,7 +48,7 @@ def _get_supplierinfo_pricelist_price(
             if seller and seller.product_uom != price_uom:
                 price = seller.product_uom._compute_price(price, price_uom)
 
-            convert_to_price_uom = lambda price: self.uom_id._compute_price(
+            convert_to_price_uom = lambda price: self.uom_id._compute_price(  # noqa: E731
                 price, price_uom
             )
```

Comment:
Don't sweep under the carpet :wink: You can make an internal method to achieve the same.

```suggestion
            def convert_to_price_uom(price, price_uom=price_uom):
                return self.uom_id._compute_price(price, price_uom)
```

Anyway, maybe it's pointless to don't call it directly anyway